### PR TITLE
Update configure-object-storage-backend.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
#### What this PR does
Adds a note about how to use the environment variables.

#### Which issue(s) this PR fixes or relates to

https://github.com/grafana/mimir/issues/4228

Fixes #<issue number>

#### Checklist

- [x] Tests updated.
- [x] Documentation added.
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [x] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: docs/sources/mimir/configure/configure-object-storage-backend.md
Comment (jdbaldry) [inline docs/sources/mimir/configure/configure-object-storage-backend.md:39]:
```suggestion
To use environment variables in the configuration file, ensure that you [enable expansion]({{< relref "./configuration-parameters#use-environment-variables-in-the-configuration" >}}) for the variables.
```

**Review by jdbaldry — APPROVED** [review]
Nice! Docs LGTM, thank you.

I'll defer to a code maintainer to double check and merge :)

**Review by dimitarvdimitrov — APPROVED** [review]
thank you @Rajakavitha1! LGTM

**Pull request merged**
